src/System.Windows.Forms/src/System/Windows/Forms/ListView.cs

```diff
@@ -3353,8 +3353,8 @@ private void ForceCheckBoxUpdate()
             //
             if (CheckBoxes && IsHandleCreated)
             {
-                SendMessage((int)LVM.SETEXTENDEDLISTVIEWSTYLE, NativeMethods.LVS_EX_CHECKBOXES, 0);
-                SendMessage((int)LVM.SETEXTENDEDLISTVIEWSTYLE, NativeMethods.LVS_EX_CHECKBOXES, NativeMethods.LVS_EX_CHECKBOXES);
+                User32.SendMessageW(this, User32.WindowMessage.LVM_SETEXTENDEDLISTVIEWSTYLE, (IntPtr)LVS_EX.CHECKBOXES, IntPtr.Zero);
```

Comment:
This is not correct
```suggestion
                User32.SendMessageW(this, (User32.WindowMessage)LVM.SETEXTENDEDLISTVIEWSTYLE, (IntPtr)LVS_EX.CHECKBOXES, IntPtr.Zero);
```

Applies to other places as well.

Reply:
I see some other places in master that we use User32.WindowMessage.A_BC instead of (User32.WindowMessage)A.BC. Should we change all of them in a separate PR?

Reply:
We have duplicate and in some places triplicate definitions that need to be clean up (e.g #2077)